android/src/main/java/com/terveystalo/react_native/matomo_sdk/RNMatomoSdkModule.java

```diff
@@ -19,4 +32,57 @@ public RNMatomoSdkModule(ReactApplicationContext reactContext) {
   public String getName() {
     return "RNMatomoSdk";
   }
+
+  @ReactMethod
+  @SuppressWarnings("unused")
+  public void initialize(String apiUrl, int siteId, Promise promise) {
+    try {
+      if (tracker == null) {
+        tracker = TrackerBuilder
+                .createDefault(apiUrl, siteId)
+                .build(Matomo.getInstance(this.reactContext));
+
+        // Disable default automatic user ID generation for consistency with iOS
+        tracker.setUserId(null);
+      }
+      promise.resolve(null);
+    } catch(Exception e) {
+      promise.reject(e);
+    }
+  }
+
+  @ReactMethod
+  @SuppressWarnings("unused")
+  public void trackView(ReadableArray route, Promise promise) {
+    try {
+      List<String> routeList = new LinkedList<>();
```

Comment:
I think `ArrayList<>` would suffice. Not that it matters that much...

Reply:
https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableArray.java#L31 ?

Reply:
Hmm, that would be handy. For some reason gradle grabbed some rather old version of react-native for the dependency. Using newer versions of react in development would probably be easier after https://github.com/terveystalo/react-native-matomo-sdk/pull/2. So maybe I'll fix this later instead.